stubs/python-dateutil/dateutil/parser/__init__.pyi

```diff
@@ -35,13 +37,30 @@ class parser:
         timestr: _FileOrStr,
         default: datetime | None = ...,
         ignoretz: bool = ...,
-        tzinfos: Mapping[str, tzinfo] | None = ...,
+        tzinfos: _TzInfo | None = ...,
+        *,
+        dayfirst: bool | None = ...,
+        yearfirst: bool | None = ...,
+        fuzzy: bool = ...,
+        fuzzy_with_tokens: bool = ...,
         **kwargs: Any,
     ) -> datetime: ...
 
 DEFAULTPARSER: parser
 
-def parse(timestr: _FileOrStr, parserinfo: parserinfo | None = ..., **kwargs: Any) -> datetime: ...
+def parse(
+    timestr: _FileOrStr,
+    parserinfo: parserinfo | None = ...,
+    *,
+    dayfirst: bool | None = ...,
+    yearfirst: bool | None = ...,
+    ignoretz: bool = ...,
+    fuzzy: bool = ...,
+    fuzzy_with_tokens: bool = ...,
+    default: datetime | None = ...,
+    tzinfos: _TzInfo | None = ...,
+    **kwargs: Any,
```

Comment:
```suggestion
```

Reply:
(Technically someone might change what `DEFAULTPARSER` is and that's why we might need `**kwargs`, but we can ignore it probably). https://github.com/dateutil/dateutil/blob/master/src/dateutil/parser/_parser.py#L1368